priv/repo/migrations/20170515154812_rename_user_github_id_to_github_auth_token.exs

```diff
@@ -0,0 +1,7 @@
+defmodule CodeCorps.Repo.Migrations.RenameUserGithubIdToGithubAuthToken do
+  use Ecto.Migration
+
+  def change do
+    rename table(:users), :github_id, to: :github_auth_token
```

Comment:
Are you no longer expecting to track the GitHub user ID? I would think this would be important information for us to have on-hand and indexed.